Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -113,17 +118,22 @@ class FakespotViewController: UIViewController, Themeable {
     }
 
     func applyTheme() {
-        let colors = themeManager.currentTheme.colors
-        titleLabel.textColor = colors.textPrimary
-        errorCardView.applyTheme(theme: themeManager.currentTheme)
-        reliabilityCardView.applyTheme(theme: themeManager.currentTheme)
-        loadingView.applyTheme(theme: themeManager.currentTheme)
+        let theme = themeManager.currentTheme
+
+        view.backgroundColor = theme.colors.layer1
+        titleLabel.textColor = theme.colors.textPrimary
+
```

Comment:
Will those colors change? Do they need update from the update colors task FXIOS-7154? Just asking in case we should put a comment there so we don't miss those

Reply:
No, these colors should be fine.